### PR TITLE
Remove fabric related code from Python bindings

### PR DESCRIPTION
**Pull request by rdettai**
This PR cleans up legacy code and fixes the logger:
- remove argparse wich was not appropriate as this is a library
- set `Logger.propagate=False` to avoid inheriting configurations from root logger

## TODO

- [x] Better document the asyncio utils
- [x] Add some docs about the configs

### CODE REVIEW

File: python/vast/utils/asyncio.py [hunk 1]
```diff
@@ -3,7 +3,7 @@
 
 import vast.utils.logging
 
-logger = vast.utils.logging.get()
+logger = vast.utils.logging.get(__name__)
```
Comment (mavam) [inline python/vast/utils/asyncio.py:6]: Have you checked that this is basically `vast`?

File: python/vast/vast/vast.py [hunk 2]
```diff
@@ -13,7 +13,7 @@
 
 from .cli import CLI
 
-logger = vast.utils.logging.get("vast.vast")
+logger = vast.utils.logging.get(__name__)
```
Comment (mavam) [inline python/vast/vast/vast.py:16]: Same as above :-)

**Review by mavam — APPROVED** [review]
Glad to see this go (for now).

You may want to look at the CI and packaging scripts that still have code that installs all extras via poetry.

Note to self: I'll still document the asyncio utils.

File: python/vast/vast/vast.py (hunk 2 above)
Comment (rdettai) [reply python/vast/vast/vast.py:16]: `__name__` is vast.vast.vast

File: python/vast/utils/config.py [hunk 3]
```diff
@@ -1,29 +1,19 @@
...
     settings = Dynaconf(
-        settings_files=config_files,
+        settings_files=CONFIG_FILES,
         load_dotenv=True,
         envvar_prefix="VAST",
     )
     settings.validators.register(
-        Validator("fabric.logging.console_verbosity", default="debug"),
-        Validator("fabric.logging.file_verbosity", default="quiet"),
-        Validator("fabric.logging.filename", default="vast.log"),
```
Comment (rdettai) [inline python/vast/utils/config.py:16]: @mavam I am wondering about the config namespace we should use here. 
- Do we want the config to be the same as for VAST? That would mean that setting VAST_CONSOLE_VERBOSITY=debug sets the debug logging for both the bindings and the binary.
- [VAST config](https://github.com/tenzir/vast/blob/master/vast.yaml.example#L68) uses `-` and we use `_` here, should we unify this?

File: python/vast/vast/vast.py (hunk 2 above)
Comment (mavam) [reply python/vast/vast/vast.py:16]: Ooof, I don't think that's something the user would like.

File: python/vast/utils/asyncio.py (hunk 1 above)
Comment (mavam) [reply python/vast/utils/asyncio.py:25]: That looks reasonable, from a user's perspective.

File: python/vast/vast/vast.py (hunk 2 above)
Comment (mavam) [reply python/vast/vast/vast.py:16]: No, I don't know anything better. I'm just surprised that it's repeated *three times*.

File: python/vast/utils/config.py (hunk 3 above)
Comment (mavam) [reply python/vast/utils/config.py:16]: Yes, let's unify where we can.

**Pull request merged**